engine/launcher/src/main/scala/org/enso/launcher/Main.scala

```diff
@@ -170,8 +177,39 @@ object Main {
 
   private def installDistributionCommand: Subcommand[Config => Unit] =
     Subcommand("distribution") {
-      Opts.pure { (_: Config) =>
-        println(s"Install distribution")
+      val autoConfirm = Opts.flag(
+        "auto-confirm",
+        "Proceeds with installation without asking confirmation questions. " +
```

Comment:
I know you mention the effect of this flag on `bundle-install-mode` in that option's description, but I feel like the fact that the dependency exists should be surfaced in _this_ flag's documentation too.